src/components/ReviewList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import ReviewCard from "./ReviewCard";
+import { getReviews } from "../api";
+
+const ReviewList = ({ category }) => {
+  const [reviews, setReviews] = useState([]);
+  useEffect(() => {
+    getReviews().then(({ reviews }) => {
+      console.log(reviews);
+      setReviews(
+        category
+          ? reviews.filter((review) => review.category === category)
+          : reviews
+      );
+    });
```

Comment:
Could you remove this functionality for now. This is ticket 9.
Also, the way you've done it isnt quite right. When you get to the ticket - you can use your api to do the filtering. 

For this ticket, we just need all the reviews please - no filtering